library/src/main/java/com/wultra/android/sslpinning/CertStoreConfiguration.kt

```diff
@@ -186,10 +192,19 @@ class CertStoreConfiguration(
          * Fallback certificate fingerprint.
          * Useful for situations when no fingerprints has been loaded from the server yet.
          */
+        @Deprecated("Use fallbackCertificates instead. This option is ignored when fallbackCertificates method is used.")
```

Comment:
No sure if it's better to deprecate or to remove right now. The priority of the other one is a bit confusing.

Reply:
I don't want to make breaking changes right now as it's not necessary. I changed the wording at least.